nix/process-compose/default.nix

```diff
@@ -72,10 +73,9 @@ in
           mkProcessComposeWrapper
             {
               name = "${name}-test";
-              inherit (config) tui httpServer preHook postHook;
-              configFile = config.outputs.settingsTestFile;
+              inherit (config) preHook postHook;
+              arguments = config.test-arguments.output;
```

Comment:
The test process uses different arguments than the normal process. See also cli.nix:72.